### DISCUSSION TITLE
Is PyCuda available in Yocto ?

### DISCUSSION THREAD

**Discussion by andregirard37** [OE4T/meta-tegra#1274] [General]
Hello,
I am trying to add pycuda to my Yocto build. I would have thought it would be available in meta-tegra but I can't find anything online. I am building with Dunfell.

Thank you

**Comment by ichergui** [comment]
Hello @andregirard37 

Welcome to the OE4T community.

Please check our [meta-tegra-community](https://github.com/OE4T/meta-tegra-community) layer. 

You can find the `pycuda` recipe [here](https://github.com/OE4T/meta-tegra-community/blob/dunfell/recipes-devtools/python).

Let me know if you have any question

**Reply by andregirard37** [reply]
Thanks! It seems to build. By the way, a bit unrelated, but on gatesgarth, I was able to add tensorrt recipe just fine. In Dunfell, it doesn't find it. Any idea what happened to that recipe ?

Thanks!

**Reply by dwalkes** [reply]
See https://github.com/OE4T/meta-tegra/pull/799

In general for questions like these you can leverage github search.  See [this search of commits](https://github.com/search?q=repo%3AOE4T%2Fmeta-tegra+tensorrt+&type=commits) and [this search of pull requests](https://github.com/search?q=repo%3AOE4T%2Fmeta-tegra+tensorrt+&type=pullrequests)
